flixel/graphics/tile/FlxDrawTilesItem.hx

```diff
@@ -73,6 +80,11 @@ class FlxDrawTilesItem extends FlxDrawBaseItem<FlxDrawTilesItem>
 				tempFlags |= Tilesheet.TILE_RGB;
 			}
 			
+			if (hasColorOffsets)
+			{
+				tempFlags |= Tilesheet.TILE_TRANS_COLOR;
```

Comment:
Referencing this constant breaks compatibility with legacy and versions of openfl next that don't have this feature yet.